### ISSUE TITLE
Dimensionality-Driven Learning with Noisy Labels

### ISSUE THREAD

**Issue by Swall0w** [Swall0w/papers#592]
Ma, Xingjun, Wang, Yisen, Houle, Michael E., Zhou, Shuo, Erfani, Sarah M., Xia, Shu-Tao, Wijewickrema, Sudanthi, Bailey, James

Datasets with significant proportions of noisy (incorrect) class labels
present challenges for training accurate Deep Neural Networks (DNNs). We
propose a new perspective for understanding DNN generalization for such
datasets, by investigating the dimensionality of the deep representation
subspace of training samples. We show that from a dimensionality perspective,
DNNs exhibit quite distinctive learning styles when trained with clean labels
versus when trained with a proportion of noisy labels. Based on this finding,
we develop a new dimensionality-driven learning strategy, which monitors the
dimensionality of subspaces during training and adapts the loss function
accordingly. We empirically demonstrate that our approach is highly tolerant to
significant proportions of noisy labels, and can effectively learn
low-dimensional local subspaces that capture the data distribution.

https://arxiv.org/abs/1806.02612
